docs/content.md

```diff
@@ -1008,14 +1041,23 @@ If you want to move where this branch sits, click on the first title 'page title
 Select the correct place from the drop down.
 
 ## Redirect URLs
-If you search in Visual Studio Code 'redirects' it will take you to a page 'redirects.yml'
-**Currently you can only redirect internal links in this file. External redirects are handled in the routes file; you may need developer support to create or amend these.**
-You will need to follow the pattern that the list of redirects uses in this file.
-It will need to be in quotation marks. Then, you need to put the old url a colon and then the new url like this "/oldurl": "/newurl"
-Example: "/train-to-be-a-teacher/teacher-training-personal-statement": "/how-to-apply-for-teacher-training/teacher-training-personal-statement"
-Press enter to create an indention - 
-Make sure the line is indented (the fine, white line that is currently running on the left hand side of all urls).
-The easiest way to make sure you don't miss any pages to redirect is to do a search which will show you all the pages you need to include.
+
+A redirect is a way to send users to a different page to the URL they requested. We set up redirects where we do things like change the URL of a page, or remove a page. **If you need to set up a redirect from an external URL, these are handled in the routes file; you'll need developer support to create or amend these.**
+
+To set up a redirect for a Get Into Teaching page, go to the 'redirects.yml' file. This file contains a list of all existing redirects. 
+
+To make add a new redirect:
+
+1. Check if the redirect you want to add fits into an existing redirect category. If not, create a new one.
```

Comment:
Is this slightly confusing to talk about categories? Would it be better to say something like 'Redirects are grouped under headings based on which content they relate to. Check if the redirects fits into an existing heading/section (?). If not, create a new one.